### PR TITLE
[docs] Update Slack link

### PR DESCRIPTION
**Pull request by hellendag**
### Summary & Motivation

Update docs Slack links to point at dagster.io/slack.

### How I Tested These Changes

yarn dev, verify that links navigate correctly.

### CODE REVIEW

**Comment by hellendag** [comment]
Current dependencies on/for this PR:
* master
  * **PR #10695** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/10695" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/10695?utm_source=stack-comment).

**Review by rexledesma — APPROVED** [review]

File: docs/content/integrations/fivetran.mdx [hunk 1]
```diff
@@ -70,7 +70,7 @@ fivetran_instance = fivetran_resource.configured(
 )
 ```
 
-For more information on the additional configuration options available for the Fivetran resource, see the [API reference](/\_apidocs/libraries/dagster-fivetran#dagster_fivetran.fivetran_resource).
+For more information on the additional configuration options available for the Fivetran resource, see the [API reference](/_apidocs/libraries/dagster-fivetran#dagster_fivetran.fivetran_resource).
```
Comment (rexledesma) [inline docs/content/integrations/fivetran.mdx:73]: Odd that these linting errors aren't being caught by CI/CD 🤔 cc @benpankow @yuhan

**Review by rexledesma — APPROVED** [review]
thanks! the formatting errors look real, but surprised it's not being corrected by bk

File: docs/content/integrations/fivetran.mdx (hunk 1 above)
Comment (hellendag) [reply docs/content/integrations/fivetran.mdx]: Looks like maybe there's a bit of disconnect between the lint run and the mdx-format run.

**Pull request merged**